sneck/Deck.py

```diff
@@ -911,8 +976,35 @@ def update(self, board: dict) -> None:
         
         print(f'[UPDATE] Updated board {self.__id}.')
 
+    def serialize(self, complete: bool = False) -> dict:
+        if complete:
+            result = {'title': self.__title, 'owner': self.__owner, 'color': self.__color, 'archived': self.__archived, 'labels': self.__labels,
+                    'acl': self.__acl, 'permissions': self.__permissions, 'users': self.__users, 'shared': self.__shared, 'id': self.__id,
+                    'lastModified': dt.timestamp(self.__last_edited_time), 'deletedAt': self.__deletion_time,
+                    'settings': {'notify-due': self.__notifications, 'calendar': self.__synchronize} }
+        else:
+            result = {'title': self.__title, 'color': self.__color}
+
+        return result
+
     def add_card(self, sid: int, card: DeckCard) -> DeckCard:
         return self.get_stack(sid).add_card(card)
+    
+    def delete_card(self, sid: int, cid: int) -> None:
+        self.get_stack(sid).delete_card(cid)
+
+    def add_stack(self, stack: DeckStack) -> None:
+        new_stack = self.__api.post(f'boards/{self.__id}/stacks', stack.serialize())
+        # There is no response with the data we need to identify the new stack
+        
+        self.update(self.serialize())
+
+    def delete_stack(self, sid: int) -> None:
+        self.__api.delete(f'boards/{self.__id}/stacks/{sid}')
+
+        if self.get_stack(sid) is not None:
+            self.get_stack(sid).set_deletion_time(dt.timestamp(dt.now()))
```

Comment:
The deletion time is a part of the Deck API - it is better to just let the update retrieve the new deletion timestamp instead of setting it manually for better data consistency.
```suggestion
```